doc/getting_started/getting_started.rst

```diff
@@ -1,87 +1,96 @@
+`Install ROS2 Foxy <https://docs.ros.org/en/foxy/Installation.html>`_.
+It is easy to miss steps when going through the ROS2 installation tutorial. If you run into errors in the next few steps, a good place to start is to go back and make sure you have installed ROS2 correctly.
+
+Install `rosdep2 <http://wiki.ros.org/rosdep>`_ to install system dependencies : ::
+
+  sudo apt install python3-rosdep2 
 
-Once you have ROS installed, make sure you have the most up to date packages: ::
+Once you have ROS2 installed, make sure you have the most up to date packages: ::
 
   rosdep update
   sudo apt update
   sudo apt dist-upgrade
 
-Install `catkin <http://wiki.ros.org/catkin>`_ the ROS build system: ::
+Install `Colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-Tutorial.html#install-colcon>`_ the ROS2 build system: ::
 
-  sudo apt install ros-noetic-catkin python3-catkin-tools
+  sudo apt install python3-colcon-common-extensions
 
-Install `wstool <http://wiki.ros.org/wstool>`_ : ::
+Install `vcstool <https://index.ros.org/d/python3-vcstool/>`_ : ::
 
-  sudo apt install python3-wstool
+  sudo apt install python3-vcstool
 
-Create A Catkin Workspace and Download MoveIt Source
+Create A Colcon Workspace and Download MoveIt2 Source
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-These tutorials rely on the master branch of MoveIt, which requires a build from source.
-You will need to have a `catkin <http://wiki.ros.org/catkin>`_ workspace setup: ::
-
-  mkdir -p ~/ws_moveit/src
-  cd ~/ws_moveit/src
-
-  wstool init .
-  wstool merge -t . https://raw.githubusercontent.com/ros-planning/moveit/master/moveit.rosinstall
-  wstool update -t .
-
-Download Example Code
-^^^^^^^^^^^^^^^^^^^^^
-
-To easily follow along with these tutorials, you will need a **ROBOT_moveit_config** package. The default demo robot is the Panda arm from Franka Emika. To get a working **panda_moveit_config** package, we recommend you install from source.
-
-Within your `catkin <http://wiki.ros.org/catkin>`_ workspace, download the tutorials as well as the ``panda_moveit_config`` package: ::
-
-  cd ~/ws_moveit/src
-  git clone https://github.com/ros-planning/moveit_tutorials.git -b master
-  git clone https://github.com/ros-planning/panda_moveit_config.git -b melodic-devel
+These tutorials rely on the master branch of MoveIt2, which requires a build from source.
+You will need to have a `colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-Tutorial.html#install-colcon>`_ workspace setup: ::
 
-.. note:: For now we will use a pre-generated ``panda_moveit_config`` package but later we will learn how to make our own in the `MoveIt Setup Assistant tutorial <../setup_assistant/setup_assistant_tutorial.html>`_.
+  mkdir -p ~/ws_moveit2/src
+  cd ~/ws_moveit2/src
 
+  wget https://raw.githubusercontent.com/ros-planning/moveit2/main/moveit2.repos
+  vcs import < moveit2.repos
```

Comment:
You also need to pull down this repo and its dependencies. 

> wget https://raw.githubusercontent.com/ros-planning/moveit2_tutorials/main/moveit2_tutorials.repos
vcs import < moveit2_tutorials.repos

I made a simple version of building from source on the [README](https://github.com/ros-planning/moveit2_tutorials#moveit2-tutorials-source-build)